addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -426,8 +426,9 @@ var FileWidget = SearchableMediaWidget.extend({
 
         // adapt load more
         var noMoreImgToLoad = this.numberOfAttachmentsToDisplay >= this.attachments.length;
-        this.$('.o_load_more').toggleClass('d-none', noMoreImgToLoad);
-        this.$('.o_load_done_msg').toggleClass('d-none', !noMoreImgToLoad);
+        var $loadMoreButton = this.$('.o_load_more');
+        this.$('.o_load_done_msg').toggleClass('d-none', $loadMoreButton.hasClass('d-none') || !noMoreImgToLoad);
+        $loadMoreButton.toggleClass('d-none', noMoreImgToLoad);
```

Comment:
@fja-odoo It my not written like that before I reviewed right ? I find it really difficult to understand what this does. The new state depends on the old state + some condition... while it should only depend on the number of total images and the number of displayed images.

Reply:
Previously it was only depending on the  number of total images and the number of displayed images but the new specs ask to only display the message if the "Load more" button was previously displayed.

Reply:
Can't the condition be replaced by `nbDisplayed <= minNbDisplayed` ?